app/models/ecf_participant_eligibility.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class ECFParticipantEligibility < ApplicationRecord
+  belongs_to :participant_profile, class_name: "ParticipantProfile::ECF", touch: true
+  before_validation :determine_status
```

Comment:
has papertrail? or do we not expect this to be edited?

Reply:
> We're going to need to keep date of birth on TeacherRecord for the time being, let's ignore it from papertrail `has_paper_trail ignore: [:title, :rating]` https://github.com/paper-trail-gem/paper_trail#2c-choosing-attributes-to-monitor
Why not just keep the validation data model until we're done with it instead?  I've currently built it so that it does that (but wouldn't if the status was `:eligible`)

Reply:
How do we distinguish between someone who has a validated TRN and is in the holding pen and someone who needs manual TRN checks?

Reply:
Under holding pen rules:
Validated TRN and eligible => status is `:matched`, `qts` is true
Validated TRN and not eligible => status is `:matched`, `qts` is false OR status is `:manual_check`
Validated TRN and existing different TRN on => status is `:manual_check`
TRN not validated => no `ECFParticipantEligibility` record